static/.htaccess

```diff
@@ -112,10 +112,10 @@ RewriteRule ^docs/apis-clients/working-with-apis-clients/?$ /docs/apis-tools/wor
 RewriteRule ^docs/apis-clients(.*)$ /docs/apis-tools$1 [R=301,L]
 RewriteRule ^optimize/apis-clients(.*)$ /optimize/apis-tools$1 [R=301,L]
 RewriteRule ^docs/8.0/apis-clients(.*)$ /docs/8.0/apis-tools$1 [R=301,L]
-RewriteRule ^optimize/3.8.0/apis-clients(.*)$ /optimize/3.8.0/apis-tools$1 [R=301,L]
-RewriteRule ^docs/1.3/apis-clients(.*)$ /docs/1.3/apis-tools$1 [R=301,L]
+RewriteRule ^optimize/3.8.0/apis-clients(.*)$ /optimize/3.8.0/apis-tools$1 [R=301,L] -->
+RewriteRule ^docs/apis-clients(.*)$ /docs/apis-tools$1 [R=301,L]
```

Comment:
The more important thing to notice here is that the rules that were created for 1.3 are redefined, without the version in the URL.